rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2018-10-15
+nightly-2018-10-30
```

Comment:
why not `nightly-2018-11-08` (let's use the same compiler version across all pwasm-* crates, to reduce confusion in case of bugs in the compiler itself)?

Reply:
This is going to happen once `pwasm-std` version `0.12` is being used. ;)